packages/create-block/lib/templates/esnext/src/edit.js.mustache

```diff
@@ -5,6 +5,11 @@
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+
```

Comment:
Why did you choose to import the `editor.scss` into the edit file? I think this could make it a bit more confusing since where you import it doesn't change anything and it it were added to the import statement inside the `index.js` file it would make more sense in my mind. Curios about how you feel on this @gziolo  :)

Reply:
Should I rename the file to `edit.scss` maybe? My idea was, that you create a file that targets the code. It's not obvious since it's using the class that is automatically generated and injected in `edit` with:

```js
<p className={ className }>
```